book/scheduling-and-threading.md

```diff
@@ -336,12 +488,13 @@ requestAnimationFrame(callback);
 ```
 
 This code will do two things: request an *animation frame* (rendering) task to
-be run on the event loop,[^animation-frame] and call `callback` at the
-beginning of that rendering task. This is super useful to web page authors, as
-it allows them to do any setup work related to rendering just before it occurs.
-The implementation of this JavaScript API is straightforward: add a new dirty
-bit to `Tab` and code to call the JavaScript callbacks during the next
-animation frame.
+be scheduled on the event loop,
+[^animation-frame] and call `callback` *at the beginning* of that rendering
```

Comment:
Footnotes like this should go right after punctuation, without a space.

Reply:
Fixed in direct commit to main.